eth/eventdatahandler/task_executor_test.go

```diff
@@ -45,10 +46,99 @@ func TestExecuteTask(t *testing.T) {
 	require.NoError(t, task.Execute())
+	if logs.Len() == 0 {
+		t.Errorf("No logs")
+	} else {
+		entry := logs.All()[len(logs.All())-1]
+		require.Equal(t, "validator wasn't started", entry.Message)
+	}
+}
+
+func TestExecuteStopValidatorTask(t *testing.T) {
+	logger, logs := setupLogsCapture()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	edh, err := setupDataHandler(t, ctx, logger)
+	require.NoError(t, err)
+
+	task := NewStopValidatorTask(edh.taskExecutor, ethcommon.Hex2Bytes("b24454393691331ee6eba4ffa2dbb2600b9859f908c3e648b6c6de9e1dea3e9329866015d08355c8d451427762b913d1"))
+	require.NoError(t, task.Execute())
+	if logs.Len() == 0 {
+		t.Errorf("No logs")
+	} else {
+		entry := logs.All()[len(logs.All())-1]
+		require.Equal(t, "removed validator", entry.Message)
+	}
+}
+
+func TestExecuteLiquidateClusterTask(t *testing.T) {
+	logger, logs := setupLogsCapture()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	edh, err := setupDataHandler(t, ctx, logger)
+	require.NoError(t, err)
+	var shares []*ssvtypes.SSVShare
+	share := &ssvtypes.SSVShare{
+		Share: spectypes.Share{
+			ValidatorPubKey: ethcommon.Hex2Bytes("b24454393691331ee6eba4ffa2dbb2600b9859f908c3e648b6c6de9e1dea3e9329866015d08355c8d451427762b913d1"),
+		},
+	}
+	shares = append(shares, share)
+	task := NewLiquidateClusterTask(edh.taskExecutor, ethcommon.HexToAddress("0x0000000000000000000000000000000000000001"), []uint64{1, 2, 3}, shares)
+	require.NoError(t, task.Execute())
+	if logs.Len() == 0 {
+		t.Errorf("No logs")
+	} else {
+		entry := logs.All()[len(logs.All())-1]
+		require.Equal(t, "executed task", entry.Message)
+	}
+}
+
+func TestExecuteReactivateClusterTask(t *testing.T) {
+	logger, logs := setupLogsCapture()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	edh, err := setupDataHandler(t, ctx, logger)
+	require.NoError(t, err)
+	var shares []*ssvtypes.SSVShare
+	share := &ssvtypes.SSVShare{
+		Share: spectypes.Share{
+			ValidatorPubKey: ethcommon.Hex2Bytes("b24454393691331ee6eba4ffa2dbb2600b9859f908c3e648b6c6de9e1dea3e9329866015d08355c8d451427762b913d1"),
+		},
+	}
+	shares = append(shares, share)
+	task := NewReactivateClusterTask(edh.taskExecutor, ethcommon.HexToAddress("0x0000000000000000000000000000000000000001"), []uint64{1, 2, 3}, shares)
+	require.NoError(t, task.Execute())
+	if logs.Len() == 0 {
+		t.Errorf("No logs")
+	} else {
+		entry := logs.All()[len(logs.All())-1]
+		require.Equal(t, "executed task", entry.Message)
+	}
+}
+
+func TestExecuteUpdateFeeRecipientTask(t *testing.T) {
+	logger, logs := setupLogsCapture()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	edh, err := setupDataHandler(t, ctx, logger)
+	require.NoError(t, err)
+	task := NewFeeRecipientTask(edh.taskExecutor, ethcommon.HexToAddress("0x0000000000000000000000000000000000000001"), ethcommon.HexToAddress("0x0000000000000000000000000000000000000002"))
```

Comment:
Would `0x1`/`0x2` work? If it would, I think it could be a bit easier to read

Reply:
Ethereum address is 20 bytes hash with hex 40 chars so I prefer to use full length, but yes it will work as well